train_model.py

```diff
@@ -62,17 +63,17 @@ def main(nb_episodes: int, cpt_env: CPTEnvironment, agent: DQLAgent, training_da
 
 
 if __name__ == "__main__":
-    training_data_folder = "./data_fabian/train"
+    training_data_folder = "P:/schemagan/synthetic_database/512x32_20k/train"
     num_episodes = 100
-    actions = [5, 10, 25, 50, 100]  # actions in number of pixels
+    actions = [50, 100, 150]  # actions in number of pixels
     output_folder = "results"
 
     cpt_env = CPTEnvironment(actions,
                              max_nb_cpts=50,
                              weight_reward_cpt=0.5,
                              image_width=512,
                              max_first_step=20,
-                             interpolation_method=InverseDistance(nb_points=6),
+                             interpolation_method=SchemaGANInterpolator("P:/schemagan/model_000036.h5"),
```

Comment:
I would put the model locally and add the relative path.  at least in the commit. the same applies for the training_data_folder on line 66

Reply:
ok